Lib/test/test_dbm.py

```diff
@@ -75,10 +75,9 @@ def test_anydbm_creation(self):
     def test_anydbm_creation_n_file_exists_with_invalid_contents(self):
         # create an empty file
         test.support.create_empty_file(_fname)
```

Comment:
Please keep the empty line. IMHO it helps for the readability.

Reply:
This test is short (just 4 lines), and no other test method contains an empty line inside. Having an empty line that splits this short test rather adds a false signal that attract an attention to an insignificant detail.

But if you still think that it is worth to restore an empty line, I'll do this.